### PR TITLE
update civ

### PR DESCRIPTION
**Pull request by jabia99**
Update cloud-image-val to latest release

This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Comment by ondrejbudai** [comment]
I vote against abbreviating cloud-image-val into civ. It reminds me of the one and only civ too much. (otherwise, this looks good)

image

**Comment by jabia99** [comment]
We are running into a lot of dependency problems. It's time to containerize cloud-image-val. We'll pause this PR in the meantime

File: test/data/repositories/rhel-84.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
+{
+  "aarch64": [
+    {
+      "name": "baseos",
+      "baseurl": "https://cdn.redhat.com/content/dist/rhel8/8.4/aarch64/baseos/os",
```
Comment (thozza) [inline test/data/repositories/rhel-84.json:5]: Adding this non-eus repo is not needed. All the content from 8.4 is available in EUS.

File: test/data/repositories/rhel-84.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      "rhsm": true,
+      "check_gpg": true
+    },
+    {
+      "name": "appstream",
+      "baseurl": "https://cdn.redhat.com/content/dist/rhel8/8.4/aarch64/appstream/os",
```
Comment (thozza) [inline test/data/repositories/rhel-84.json:12]: Ditto - non-eus repos

File: test/data/repositories/rhel-84.json [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      "check_gpg": true,
+      "image_type_tags": [
+        "gce"
+      ]
+    },
+    {
+      "name": "eus",
+      "baseurl": "https://cdn.redhat.com/content/eus/rhel8/8.4/x86_64/appstream/os",
```
Comment (thozza) [inline test/data/repositories/rhel-84.json:103]: You should define the EUS repo also for BaseOS and also do it for all architectures.

**Review by thozza — COMMENTED** [review]
All looks good, but I have some comments related to the added RHEL-8.4 repos.

Non-EUS 8.4 is already EOL, so IMO it does not make sense to use those repos at all for any testing. I propose to make the added 8.4 repo definition point only to EUS URLs for all architectures and both AppStream and BaseOS.

BTW, the `cloud-image-val.py` tool is producing some warnings:
```
test_suite/generic/test_generic.py:438
  /opt/app-root/src/test_suite/generic/test_generic.py:438: PytestUnknownMarkWarning: Unknown pytest.mark.wait - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.wait(120)
../lib64/python3.9/site-packages/paramiko/transport.py:236
  /opt/app-root/lib64/python3.9/site-packages/paramiko/transport.py:236: CryptographyDeprecationWarning: Blowfish has been deprecated
    "class": algorithms.Blowfish,
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```
https://gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/-/jobs/2826714740#L3277

**Review by thozza — DISMISSED** [review]
👍

**Comment by jabia99** [comment]
As one of the eus repo is failing. The related test in CIV is being temporarily disabled for rhel 8.4

See [COMPOSER-1654](https://issues.redhat.com/browse/COMPOSER-1654) for reference

**Review by atodorov — APPROVED** [review]

**Review by thozza — APPROVED** [review]

**Pull request merged**